recipe/meta.yaml

```diff
@@ -56,3 +56,4 @@ extra:
     - fabianhofmann
     - fneum
     - xylar
+    - lkstrp
```

Comment:
Adding yourself as a maintainer will unfortunately not be useful here because we won't merge into the main branch.  You can make a separate PR to add yourself but let's get this straightened out first.